### PR TITLE
Server: Add Commit Service

### PR DESCRIPTION
**Pull request by GalCegla**
- Add get one Commit method
 - Add get many Commits method

### CODE REVIEW

File: packages/amplication-server/src/core/commit/commit.service.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from 'nestjs-prisma';
+import { FindManyCommitArgs, FindOneCommitArgs } from 'prisma/dal';
+import { Commit } from 'src/models';
+
+@Injectable()
+export class CommitService {
+  constructor(private readonly prisma: PrismaService) {}
+
+  async getCommit(args: FindOneCommitArgs): Promise<Commit> {
```
Comment (iddan) [inline packages/amplication-server/src/core/commit/commit.service.ts:10]:
```suggestion
  async findOne(args: FindOneCommitArgs): Promise<Commit> {
```

File: packages/amplication-server/src/core/commit/commit.service.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    return this.prisma.commit.findOne(args);
+  }
+
+  async getCommits(args: FindManyCommitArgs): Promise<Commit[]> {
```
Comment (iddan) [inline packages/amplication-server/src/core/commit/commit.service.ts:14]:
```suggestion
  async findMany
  (args: FindManyCommitArgs): Promise<Commit[]> {
```

**Comment by iddan** [comment]
Please add a unit test and a Nest module

**Comment by GalCegla** [comment]
Moved commit.module.ts to Commit Resolver Branch

**Pull request closed without merging**